### PR TITLE
#168057597 get transactions(fix)

### PR DESCRIPTION
**Pull request by Jaman-dedy**
## Description ##
Enable users to get properties' information when the retrieve their transactions. 
## How can this be manually tested

- Clone this Repo `https://github.com/landvilleng/landville-backend-web-api.git`
- Checkout to this `ch-fix-get-transactions-168057597`
- First, log in with Client Admin credentials and create Compony and properties
- Then, log in with 'buyer` credentials and add transactions, you can hit this endpoint `localhost:8000/api/v1/transactions/card-pin/`
- Then activate the transaction, you can hit  the point `localhost:8000/api/v1/transactions/validate-card/`
- Then get all transaction on this endpoint `localhost:8000/api/v1/transactions/`
## Type of change ##
Please select the relevant option
- [x] Bug fix(a non-breaking change which fixes an issue)
- [ ] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [ ] unit test
- [x] None

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[#168057597](https://www.pivotaltracker.com/story/show/168057597)

### CODE REVIEW

**Review by Tevinthuku — APPROVED** [review]
Works as expected. I can successfully view the property details. Nice work @Jaman-dedy

File: transactions/serializers.py [hunk 1]
```diff
@@ -60,14 +60,18 @@ class Meta:
...
+                  'address']
 
     def get_percentage_completion(self, obj):
         """
         Return the total percentage a user has so far paid for a
         property
         """
+        print('object', obj)
         request = self.context.get('request')
+        print('request', request)
```
Comment (Niyitangasam) [inline transactions/serializers.py]: Great work, I can view the property details. However, I think you forgot to remove those print you used for debugging @Jaman-dedy

**Review by joelethan — CHANGES_REQUESTED** [review]
Heyy @Jaman-dedy Could add some tests to raise the coverage. I see it dropped by 1.3%

**Review by neelxie — APPROVED** [review]
@Jaman-dedy great work, feature seems to works seamlessly.

**Review by Victoradukwu — CHANGES_REQUESTED** [review]
@Jaman-dedy Nice job. Just fix the comments below:

- Please remove the print statement as spotted by @Niyitangasam.

- Change the title of the PR to conform to the team convention.

- ~This task is also to handle the resolution of the migration issues~

- There is also a drop in the test coverage.

**Comment by Tevinthuku** [comment]
Hi @Victoradukwu , I'm getting this error whenever I try to access coveralls

image

So I've run the test coverage directly on this PR and the app @Jaman-dedy  has touched has been tested 100%

image

File: transactions/serializers.py (hunk 1 above)
Comment (Jaman-dedy) [reply transactions/serializers.py]: Thanks @samdeniyi , I worked on that

**Review by Niyitangasam — APPROVED** [review]

**Review by shemaeric — APPROVED** [review]
LGTM

**Review by Manorlds-Eaglespark — APPROVED** [review]

**Review by CryceTruly — APPROVED** [review]

**Review by wangonya — APPROVED** [review]

**Review by Genza999 — APPROVED** [review]

**Review by Victoradukwu — APPROVED** [review]

**Pull request merged**